Sources/Harmony/Data/DataSource/Async/AsyncDataSource.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  AsyncDataSource.swift
+//  Harmony
+//
+//  Created by Joan Martin on 10/6/22.
+//
+
+import Foundation
+
+@available(iOS 13.0.0, *)
+public protocol AsyncDataSource {}
+
+@available(iOS 13.0.0, *)
+public protocol AsyncGetDataSource : AsyncDataSource {
```

Comment:
Is it intentional that these classes can be subclassed (inside the same module)? Otherwise, we should make them final, since it's not the default behavior in Swift.